test/aircraft/FlightManagementSystem/Fms.spec.js

```diff
@@ -137,13 +138,25 @@ ava('.init() calls ._buildLegCollection()', (t) => {
     t.true(_buildLegCollectionSpy.calledWithExactly(ARRIVAL_AIRCRAFT_INIT_PROPS_MOCK.route));
 });
 
-ava('.setDepartureRunway() changes #departureRunway to the ', (t) => {
-    const nextRunwayMock = airportModelFixture.getRunway('19R');
+ava('.setDepartureRunway() sets #departureRunwayModel to the specified runway model', (t) => {
+    const nextRunwayFixture = airportModelFixture.getRunway('19R');
     const fms = buildFmsMockForDeparture();
 
-    fms.setDepartureRunway(nextRunwayMock);
+    fms.setDepartureRunway(nextRunwayFixture);
 
-    t.true(_isEqual(fms.departureRunwayModel, nextRunwayMock));
+    t.true(_isEqual(fms.departureRunwayModel, nextRunwayFixture));
```

Comment:
I've actually started using `t.deepEqual()` is instead of lodash. Not something you need to change but something to keep in mind.